src/instruments/src/SD/Common/Spoilers.tsx

```diff
@@ -0,0 +1,80 @@
+import { useSimVar } from '@instruments/common/simVars';
+import React from 'react';
+import { ComponentPositionProps } from './ComponentPositionProps';
+import { ComponentSidePositionProps } from './ComponentSidePositionProps';
+import { useHydraulics } from './HydraulicsProvider';
+import { HydraulicSystem } from './HydraulicSystem';
+import { SvgGroup } from './SvgGroup';
+import './Spoilers.scss';
+
+export const Spoilers = ({ x, y }) => {
+    const [aileronLeftDeflectionState] = useSimVar('AILERON LEFT DEFLECTION PCT', 'percent over 100', 50);
+    const [aileronRightDeflectionState] = useSimVar('AILERON RIGHT DEFLECTION PCT', 'percent over 100', 50);
+
+    const [leftSpoilerState] = useSimVar('SPOILERS LEFT POSITION', 'percent over 100', 50);
+    const [rightSpoilerState] = useSimVar('SPOILERS RIGHT POSITION', 'percent over 100', 50);
+    const [speedBrakeHandlePosition] = useSimVar('SPOILERS HANDLE POSITION', 'percent over 100', 100);
+
+    const [spoilersArmed] = useSimVar('L:A32NX_SPOILERS_ARMED', 'boolean', 500);
+
+    const leftSpoilerUp = leftSpoilerState > 0.1;
+    const leftAileronUp = aileronLeftDeflectionState < -0.5;
+    const rightSpoilerUp = rightSpoilerState > 0.1;
+    const rightAileronUp = aileronRightDeflectionState > 0.5;
```

Comment:
I guess -0.5 and 0.5 are good values but I prefer to ask :D

Reply:
I have no idea. These values were in the old code and this PR (mostly) tries to refactor from old to React. I'm sure these values can or should be improved. As far as I could see right now these values show the joystick deflection, not the actual deflection of the various control surfaces. So yeah, that's likely wrong 😀.